docs/installation-configuration.md

```diff
@@ -19,7 +19,7 @@ The Pulumi Equinix provider is available as a package in all Pulumi languages:
 The Equinix provider binary is a third party binary. It can be installed using the `pulumi plugin` command.
 
 ```bash
-pulumi plugin install resource equinix v0.1.0
+pulumi plugin install resource equinix --server github://api.github.com/equinix
```

Comment:
Seems accurate: 
- https://www.pulumi.com/docs/using-pulumi/pulumi-packages/how-to-author/#support-for-github-releases